src/components/ListUniqueValues.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div class="list-unique-values">
+    <div>FILTER VALUES</div>
+    <input
+      class="list-unique-values__search-box"
+      placeholder="Search 50 items"
+      @input="search($event.target.value)"
+    />
+    <div class="list-unique-values__select-clear-all">
+      <span class="list-unique-values__select-all" @click="selectAll">Select all</span> &nbsp;
+      <span class="list-unique-values__clear-all" @click="clearAll">Clear all</span>
+    </div>
+    <div
+      class="list-unique-values__checkbox-container"
+      v-for="value in searchedValues"
+      :key="value.value"
+    >
+      <CheckboxWidget
+        :label="`${value.value} (${value.nbOcc})`"
+        :value="isChecked[value.value]"
+        @input="toggleCheck(value.value)"
+      />
+    </div>
+    <div class="list-unique-values__load-more" v-if="loaded">
+      <div>List maybe incomplete</div>
+      <div @click="loadMore" class="list-unique-values__load-more-button">Load more</div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+// This type will be imported from @/lib/dataset/helpers.ts
+type ColumnValueStat = {
+  value: any;
+  nbOcc: number;
+};
+
+import CheckboxWidget from './stepforms/widgets/Checkbox.vue';
+@Component({
+  name: 'filter-value',
+  components: { CheckboxWidget },
+})
+export default class ActionMenu extends Vue {
```

Comment:
I'm confused about the name of this component:
- file name: ListUniqueValues
- name property: filter-value
- class name: ActionMenu (probably a forgotten copy-paste there :p)

If the mission of the component is to display and filter unique values of co column, its name should reflect that :)

Reply:
Oups sorry